### PR TITLE
Add storm.exec command (SYN-7651)

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/lib/stormlib/storm.py [hunk 1]
```diff
@@ -13,6 +15,57 @@
...
+
+    async def execStormCmd(self, runt, genr):
+
+        item = None
+        async for item in genr:
+
+            text = await s_stormtypes.tostr(self.opts.query)
+            query = await runt.getStormQuery(text)
+
+            async with runt.getSubRuntime(query) as subr:
```
Comment (invisig0th) [inline synapse/lib/stormlib/storm.py:56]: I'd like to make these pump through one runtime so it behaves more like it's inline. Thoughts?

**Review by invisig0th — CHANGES_REQUESTED** [review]

File: synapse/lib/stormlib/storm.py (hunk 1 above)
Comment (Cisphyx) [reply synapse/lib/stormlib/storm.py:56]: Yeah I was going back and forth on that, I can switch it to one runtime. Do we want to allow non-runtsafe queries and potentially run different storm per-node?

Comment (invisig0th) [reply synapse/lib/stormlib/storm.py:59]: Yea, unfortunately we probably need both.

Comment (Cisphyx) [reply synapse/lib/stormlib/storm.py:59]: I adjusted it to use a single runtime and just swap out the query per-node

File: synapse/lib/stormlib/storm.py [hunk 2]
```diff
@@ -13,6 +15,76 @@
...
+            extra = await self.runt.snap.core.getLogExtra(text=text)
+            stormlogger.info(f'Executing storm query via storm.exec {{{text}}} as [{self.runt.user.name}]', extra=extra)
+
+            async with runt.getSubRuntime(query) as subr:
+                async for subp in subr.execute(genr=s_common.agen(item)):
+                    yield subp
+
+                async for item in genr:
+                    text = await s_stormtypes.tostr(self.opts.query)
+                    subr.query = await runt.getStormQuery(text)
```
Comment (invisig0th) [inline synapse/lib/stormlib/storm.py:80]: This may be bypassing the runtvar loading/detection?

Comment (Cisphyx) [reply synapse/lib/stormlib/storm.py]: Good point, adjusted to re-initialize runtvars for each per-node execution

**Pull request merged**